### ISSUE TITLE
Different namn on project id and project name don't update files

### ISSUE THREAD

**Issue by mrnicon** [krabhishek/flutterflow-vscode-extension#8]
I have a project where the project id (in the url) is  nicon2-q22kwn and the project name is nicon. This result in the strange behavior that the catalogs is created/updated but the files inside is not, and if starting from a clean project, there will be no files. 
After I changed the project name to nicon2, all the files was updated.

**Comment by jvdtoorn** [comment]
Yep, there seems to be a bug in the extension. When the project name is different than the project id, the `getProjectFolder()` function returns a different folder name than what is actually created by `flutterflow_cli`, and the files aren't properly copied over.  
Would love to see a fix for this, I would definitely start using the extension! For now I'm defaulting back to using a chain of commands to automate the code download and doing a hot reload.
